resources/flows/covid-flow.js

```diff
@@ -153,14 +153,24 @@ export default [
 		requiresAnswer: [{ question: "thermometer", value: 11 }],
 		questionText: "TODO: Add numerical input",
 		questionSecondText: "",
-		iconText: ["yes", "no", "don't know"],
+		iconText: ["37.0", "37.1", "37.2", "37.3", "37.4", "37.5", "37.6", "37.7", "37.8", "37.9", "38.0"],
 		iconColors: ["fb-orange", "fb-cyan", "fb-indigo"],
 		iconImages: [
 			"images/icons/fever_high.png",
 			"images/icons/fever_no.png",
 			"images/icons/fever_unk.png",
 		],
 	},
+	{
+		name: "fever-confirm",
+		type: "icon",
+		requiresAnswer: [{ question: "fever", value: "determined by code" }],
+		questionText: "You selected xxxx",
+		questionSecondText: "Is this correct?",
+		iconText: ["yes"],
```

Comment:
I realise that having a single answer question causes the appearance to look a bit asymmetric. You don't need to implement this, but could you explore what it would be like having three options. The first two are blank and the third is "yes". I think this would center the "yes in the middle top"

So it might look like
```js
iconText: ["", "", "yes"],
iconColors: ["fb-black", "fb-black", "fb-green"],
icomImages:["images/icons/black.png", "images/icons/black.png", "images/icons/prefer_cold.png"]
```

black.png doesn't exist so you will need to make that. 

if this doesn't work, then don't worry. Functionality wise it's good as it is :) We can add this as an issue for a future PR